pyrule_compendium/__init__.py

```diff
@@ -102,4 +113,31 @@ def get_image(self, entry: types.entry) -> objects.entry_image:
             - type: `objects.entry_image`
         """
 
-        return objects.entry_image(self.get_entry(entry), self.api)
\ No newline at end of file
+        if self._is_master_mode_entry(entry): return objects.entry_image(self.get_entry(entry), self.master_api)
+        else: return objects.entry_image(self.get_entry(entry), self.api)
```

Comment:
Again, two requests.

Reply:
I can optimize `_is_master_mode_entry` to use local data instead of a request. Otherwise, there isn’t a way to have only one request.